tasks/eva_3033/detect_project_status.py

```diff
@@ -0,0 +1,228 @@
+            "join file f on f.file_id=af.file_id "
+            f"where f.file_type='VCF' and pa.project_accession='{self.project}'"
+            "order by pa.project_accession, pa.analysis_accession")
+        filenames = []
+        current_analysis = current_assembly = current_tax_id = None
+        for project, analysis, assembly, tax_id, filename in get_all_results_for_query(self.metadata_conn, query):
+            if analysis != current_analysis:
+                if current_analysis:
+                    yield current_analysis, current_assembly, current_tax_id, filenames
+                current_analysis = analysis
+                current_assembly = assembly
+                current_tax_id = tax_id
+                filenames = []
+            filenames.append(filename)
+        yield current_analysis, current_assembly, current_tax_id, filenames
+
+    def get_taxonomy_for_project(self):
+        taxonomies = []
+        query = f"select distinct taxonomy_id from evapro.project_taxonomy where project_accession='{self.project}'"
+        for tax_id, in get_all_results_for_query(self.metadata_conn, query):
+            taxonomies.append(tax_id)
+        if len(taxonomies) == 1:
+            return taxonomies[0]
+        else:
+            logger.error(f'Cannot retrieve a single taxonomy for project {self.project}. Found {len(taxonomies)}.')
+
+    def check_accessioning_was_done(self, analysis, filenames):
+        """
+        Check that an accessioning file can be found in either noah or codon (assume access to both filesystem)
+        It parses and provide a 1000 submitted variant accessions from that project.
+        """
+        accessioning_reports = self.get_accession_reports_for_study()
+        accessioned_filenames = [self.get_accession_file(f) for f in filenames]
+        if not accessioning_reports:
+            return 0, []
+        if len(accessioning_reports) == 1:
+            accessioning_report = accessioning_reports[0]
+        elif len([r for r in accessioning_reports if r in accessioned_filenames]) == 1:
+            accessioning_report = [r for r in accessioning_reports if os.path.basename(r) in accessioned_filenames][0]
+        elif len([r for r in accessioning_reports if analysis in r]) == 1:
+            accessioning_report = [r for r in accessioning_reports if analysis in r][0]
+        elif accessioning_reports:
+            logger.warning(f'Assume all accessioning report are from project {self.project}:{analysis} and only use the first one.')
+            accessioning_report = accessioning_reports[0]
+        else:
+            logger.error(f'Cannot assign accessioning report to project {self.project} analysis {analysis} for files {accessioning_reports}')
+            accessioning_report = None
+        if not accessioning_report:
+            return 0, []
+        return self.get_accessioning_info_from_file(accessioning_report)
```

Comment:
`get_accessioning_info_from_file` currently returns just the list of ssIDs, not both count + list as the other branches do.